api/cpp/cbindgen.rs

```diff
@@ -725,6 +725,7 @@ pub struct EnabledFeatures {
     pub experimental: bool,
     pub backend_qt: bool,
     pub std: bool,
+    pub renderer_software: bool,
```

Comment:
Perhaps we need also the Skia renderer here and ifdefs in C++.

Reply:
Yes, we will need that as well.   
I'll do that when i get rid of the experimental feature.